kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/utils/DeploymentUtils.java

```diff
@@ -235,6 +241,31 @@ public static void registryCredentialsSecret(String namespace) {
         }
     }
 
+    /**
+     * Collect cluster info.
+     *
+     * @param namespace the namespace
+     * @param testClassName the test class name
+     * @param testMethodName the test method name
+     */
+    public static void collectClusterInfo(String namespace, String testClassName, String testMethodName) {
+        String formattedDate = DateTimeFormatter.ofPattern("yyyy_MM_dd_HH_mm_ss")
+                .withZone(ZoneOffset.UTC)
+                .format(Instant.now());
+        List<String> executableCommand = List.of(cmdKubeClient(namespace).toString(), "cluster-info", "dump",
+                "--namespaces", String.join(",", namespace, Constants.KAFKA_DEFAULT_NAMESPACE),
+                "--output-directory", Environment.DUMP_LOGS + "/" + testClassName.replace(".", "_") + "_" +
+                        testMethodName + "_cluster_dump_" + formattedDate);
+        ExecResult result = Exec.exec(null, executableCommand, Duration.ofSeconds(20), true, false, null);
+        if (result.isSuccess()) {
+            LOGGER.atInfo().log(result.out());
```

Comment:
I haven't run the PR but if its worked would it not just be better to log `cluster info written to XXXX` ?

Reply:
This is just what the output of `kubectl cluster-info dump` is writing in the `result.out`

Reply:
Ok but I'm not a big fan on depending on the output of another tool and assuming it will always be what we want.

Reply:
```suggestion
            LOGGER.atInfo().log("cluster-info dump written to : {}", Environment.CLUSTER_DUMP_DIR + "/" + testClassName.replace(".", "_") + "_" +
                        testMethodName + "_cluster_dump_" + formattedDate);
           LOGGER.asTrace().log("cluster-info output: {}", result.out());
```